metrics/web/test/common/presentation/metrics_theme/state/theme_notifier_test.dart

```diff
@@ -7,62 +8,112 @@ import 'package:test/test.dart';
 
 void main() {
   group("ThemeNotifier", () {
+    const lightTheme = ThemeType.light;
+    const darkTheme = ThemeType.dark;
+
     test(
-      "creates an instance with the default is dark value if the brightness is not specified",
+      "creates an instance with the light selected theme if the brightness is not specified",
```

Comment:
It seems like we should create an instance with the dark selected theme if the brightness is not specified.